src/test/java/org/assertj/core/internal/floats/Floats_assertIsCloseTo_Test.java

```diff
@@ -75,6 +79,23 @@ void should_pass_if_actual_and_expected_are_NaN() {
 
   // error or failure
 
+  @Test
+  void should_fail_if_actual_is_bigger_than_expected_byLessThan_range1(){
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> floats.assertIsCloseTo(someInfo(), 1.0f, 1.1f,
+      byLessThan(0.1f)));
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> floats.assertIsCloseTo(someInfo(), 1.0f, 1.1f,
+      within(0.0999999f)));
+  }
+
+  @Test
+  void should_fail_if_actual_is_bigger_than_expected_byLessThan_range2(){
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> floats.assertIsCloseTo(someInfo(), 0.375f, 0.125f,
+      byLessThan(0.25f)));
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> floats.assertIsCloseTo(someInfo(), 0.375f, 0.125f,
+      within(0.2499999f)));
+  }
```

Comment:
replace `assertThatExceptionOfType(AssertionError.class).isThrownBy` by `expectAssertionError` as indicated in https://github.com/assertj/assertj-core/blob/main/CONTRIBUTING.md and do that everywhere you used it.

Reply:
Oh. I forgot this! I will fix this soon! Thx!